ra/fsp/ra/fsp/src/bsp/mcu/ra8m1/bsp_feature.h

```diff
@@ -0,0 +1,448 @@
+/*
```

Comment:
I think it would be better to include all files licensed as BSD here as well.
(I think it would be desirable to only update when a new version of fsp is released.)

Reply:
@soburi : Did you mean we create only one file and describe information about license (BSD) for all?

Reply:
No, it's not.

I think it is good to add other SoC files for future support, not only RA8M1.
The fsp includes files other than BSD, so I wrote it with a meaning of other than that.
(I want to use it for migrating RA4M1.)

Zephyr convention requires an SPDX identifier for all files.
So, All files probably have license notices.

Reply:
@soburi : (Cc @KhiemNguyenT):  We have plan to support RA4M1 in 2nd half of this year. So, drivers for RA4M1 will come soon. We are looking forward to your contribution at that time too.

>Zephyr convention requires an SPDX identifier for all files.
>So, All files probably have license notices.

Thanks for your reminding me. I nearly forgot.